mesh/mesh_initializer.py

```diff
@@ -0,0 +1,40 @@
+__doc__ = """ Mesh Initializer using Pyvista """
+
+import pyvista as pv
+import numpy as np
+
+
+class MeshInitialize():
```

Comment:
why call it `MeshInitialize` and not just `Mesh`?

Reply:
The names are truly subjective, I can change it to Mesh if that's better.